### ISSUE TITLE
question about finetune batchnorm layer

### ISSUE THREAD

**Issue by clytieai** [tensorflow/models#4288]
If i use 8 pieces of GPUs and each batchsize is 2, are the parametres of BN  finetuned by  using 16 images or just 2 images each time ?

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by clytieai** [comment]
i didn't write custom code, just modify the train.sh provided by the author to train my dataset; 
the gpu is Tesla M40;
 the version of tensorflow is 1.6.0

**Comment by bitfort** [comment]
Can you clarify which model you are using?

**Comment by bitfort** [comment]
Sorry for the delay on this, but adding someone who may be able to help.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
